app/models/group.rb

```diff
@@ -1,2 +1,5 @@
 class Group < ApplicationRecord
+  belongs_to :owner, class_name: 'User', foreign_key: 'owner_id'
+
+  validates :name, :owner_id, presence: true
```

Comment:
Hey @wfbraga,

I think you can remove the `owner_id` from this validation because the `belongs_to` relation has an [:optional](https://guides.rubyonrails.org/v6.0/association_basics.html#optional) property with `false` as default value, which means that the presence of `owner` is mandatory by default.

```suggestion
  validates :name, presence: true
```

You can check this locally by creating a group without an owner, and you should see duplicated errors:

```ruby
2.7.1 :015 > group.owner
 => nil 
2.7.1 :016 > group.valid?
 => false 
2.7.1 :017 > group.errors.full_messages
 => ["Owner must exist", "Owner can't be blank"]
```

If you apply the suggested change then we are going to have this result:

```ruby
2.7.1 :003 > group.owner = nil
2.7.1 :004 > group.valid?
 => false 
2.7.1 :005 > group.errors.full_messages
 => ["Owner must exist"]
```

Reply:
I've have tested your suggestion and it worked as you shown.
Also, reading the code it seamed out of place.
I've decided commit the changes direct from you suggestion.